kotlin/src/main/kotlin/org/jdbi/v3/core/kotlin/KotlinMapper.kt

```diff
@@ -134,7 +134,7 @@ class KotlinMapper(val kClass: KClass<*>, private val prefix: String = "") : Row
         }
 
         val memberPropertyMappers = memberProperties.filter {
-            !(it.javaField?.getAnnotation(Unmappable::class.java)?.value ?: false)
```

Comment:
can't we use the helper here as well?    `JdbiAnnotations.isBound(it.javaField?)` should work, should it not? We can skip this for now if Eclipse does not like kotlin.